pkg/kube/client.go

```diff
@@ -95,9 +94,6 @@ func getConfig() (*rest.Config, error) {
 			return nil, err
 		}
 	}
-	if err != nil {
```

Comment:
This if condition seems to be necessary to return the error when [this if](https://github.com/stakater/Reloader/pull/152/files#diff-a483d6c0863082c3df21f4aad513afe2R86) condition fails and it also does not go to else.

Reply:
If `err` in `if _, err := os.Stat(kubeconfigPath); err == nil {` is not `nil`, I think it will surely go to else block. And I make a test to confirm this.

Test：
```go
func main() {
    if err := errors.New("this is an error"); err == nil {
        fmt.Printf("go if...")
    } else {
        fmt.Println("go else...")
    }
}
```

Output：
```
# apple @ liuming-dev in ~/Workspace4go/src/go-samples [9:35:45] 
$ go run library/syntax/if_else.go 
go else...
```

Reply:
@liuming-dev You are right. Thank you for testing this.